.circleci/config.yml

```diff
@@ -13,24 +13,18 @@ jobs:
       - restore_cache:
           key: node-v2-{{ .Branch }}-{{ checksum "package-lock.json" }}
       - run: npm ci
-#       - run: npm run cy:verify
       - save_cache:
           paths:
             - ~/.npm
-#             - ~/.cache
           key: node-v2-{{ .Branch }}-{{ checksum "package-lock.json" }}
-#       - run: 
-#           command: npm start
-#           background: true
-#       - run: npm run cytest
-#   deploy:
-#     executor: heroku/default
-#     steps:
-#       - checkout
-#       - heroku/install
-#       - heroku/deploy-via-git
+  deploy:
+    executor: heroku/default
+    steps:
+      - checkout
+      - heroku/install
+      - heroku/deploy-via-git
 workflows:
-  build_test:
+  build_test_deploy:
     jobs:
       - install-node
       - cypress/install
```

Comment:
nice, so you've already added Cypress as the testing suite?

Reply:
as far as I can tell as well, just gotten a brief overview of cypress testing from Thao and some blogs. The circle cypress orb is handling installing and running, hopefully cacheing as well.